### DISCUSSION TITLE
GeoJSON 2D or 3D bbox

### DISCUSSION THREAD

**Discussion by guimochila** [maplibre/maplibre-gl-js#1893] [Q&A] [answered]
In the type **MapOptions**, the option "bounds" and "maxBounds" are the type of **LngLatBoundsLike**. The type **LngLatBoundsLike** accepts a 2D bbox: 

```ts
export declare type LngLatBoundsLike = LngLatBounds | [
	LngLatLike,
	LngLatLike
] | [
	number,
	number,
	number,
	number
];
```

Reading the RFC of GeoJSON I can see BBox can be of type 2D or 3D: [Bounding Box](https://www.rfc-editor.org/rfc/rfc7946.html#section-5) 

Is there any reason why Maplibre does not support 3D in these options? (I am still new learning about maps).

Thank you.

**Comment by HarelM** [comment, marked as answer]
MapLibre is mainly 2D in terms of supported geojson. In the future, if and when Maplibre will support 3D geojson this might be relevant I guess...

**Reply by guimochila** [reply]
Thank you @HarelM!
